### DISCUSSION TITLE
Help me install on nvidia gtx 1650, no ray tracing

### DISCUSSION THREAD

**Discussion by mimic9x** [sultim-t/xash-rt#135] [Q&A] [answered]
When I run xash3d.exe, it says: '' Can't find physical device with ray tracing support ''. Is it possible to support nvidia gtx 1650, can the graphics quality be reduced?

**Comment by jurand-cpu** [comment, marked as answer]
> When I run xash3d.exe, it says: '' Can't find physical device with ray tracing support ''. Is it possible to support nvidia gtx 1650, can the graphics quality be reduced?

i saw a reply to another question, that rtx 2060 is the lowest available spec to run the mod. it makes no sense to run ray tracing mod on non ray-trace capable card, as it would run propably in single digit fps.
